warehouse/templates/accounts/login.html

```diff
@@ -11,16 +11,28 @@
     {% endif %}
 
     {% if form.errors.__all__ %}
-    <ul class="errors">{% for error in form.errors.__all__ %}<li>{{ error }}</li>{% endfor %}</ul>
+    <ul class="errors">
+      {% for error in form.errors.__all__ %}
+      <li>{{ error }}
```

Comment:
@di  I agree with @sigmavirus24 
I know that everyone should have read the HTML specification, but omitting `</li>` seems to me a special case and the content of `{{ error }}` can be a HTML.
by conclusion, I think in this case the `Special cases aren't special enough to break the rules.`